### ISSUE TITLE
Compare Row Contents

### ISSUE THREAD

**Issue by mikenac** [MrPowers/spark-fast-tests#109]
I would love to see something that allows for the comparison of the data from a single row in one frame to the data from a single row in another frame. This is especially important when comparing very wide frames. For instance, comparing the string representation of 60 columns to find out what is different is very tedious.

**Comment by wrschneider** [comment]
Agree, the approach of printing the whole row doesn't work well with wide dataframes.

It's really hard to spot the one column that's different out of 60
